packages/ember-runtime/lib/mixins/target_action_support.js

```diff
@@ -24,6 +25,18 @@ doing more complex event handling in Components.
 */
 export default Mixin.create({
   target: null,
+  targetObject: descriptor({
+    configurable: true,
+    enumerable: false,
+    get() {
+      _deprecateTargetObject();
```

Comment:
Unfortunately, I think we actually want to duplicate the code for this helper method so that it can properly be stripped from production builds.

Can you move the code into both the `get` and `set` directly?